bin/metric-postgres-dbsize.rb

```diff
@@ -79,23 +82,24 @@ class PostgresStatsDBMetrics < Sensu::Plugin::Metric::CLI::Graphite
          default: nil
 
   include Pgpass
+  include Pgdatabases
 
   def run
     timestamp = Time.now.to_i
     pgpass
+    databases = pgdatabases
     con = PG.connect(host: config[:hostname],
-                     dbname: config[:database],
+                     dbname: databases.first,
```

Comment:
I think this still needs to be updated right?

Reply:
Updated in which way? It connects to the first DB and then queries the information for all databases. There is no need to connect to each DB individually.